pages/create-collection/index.js

```diff
@@ -0,0 +1,96 @@
+import CreateNav from "@/components/CreateNav";
+import { useState } from "react";
+import styled from "styled-components";
+import Button from "@/components/Button";
+
+const StyledHeadline = styled.h1`
+  display: flex;
+  justify-content: center;
+`;
+
+const StyledFormContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100vw;
+  padding: 3em;
+`;
+
+const StyledForm = styled.form`
+  display: flex;
+  flex-direction: column;
+  width: 250px;
+  gap: 10px;
+`;
+
+const StyledColors = styled.div`
+  width: 24px;
+  height: 24px;
+  border-radius: 50px;
+  cursor: pointer;
+`;
+
+const StyledColorPicker = styled.div`
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin-top: 50px;
+`;
+
+export default function CreateCollection({ onSubmitCollection }) {
+  const [collectionColor, setCollectionColor] = useState("#ffffff");
+  const initialColors = ["#c28375", "#DAF7A6", "#75c297", "#7597c2", "#ab75c2"];
+
+  function handleCancel(event) {
+    event.preventDefault();
+    event.target.form.reset(); // reset the form
+  }
```

Comment:
As this is resetting the form and not canceling the action, I suggest to rename this function.

```suggestion
  function handleReset(event) {
    event.preventDefault();
    event.target.form.reset(); // reset the form
  }
```

I also noticed that the initial `collectionColor` is white but with resetting the form, the color in the input does not reset to white. You could reset the state to it's initial value in the `handleReset` function.